### PR TITLE
Add PT + TF automatic builds

### PR DESCRIPTION
**Pull request by LysandreJik**
Adds docker containers for both PyTorch and TensorFlow on which the pipelines will run.

Solves the dependencies errors for PyTorch and TensorFlow pipelines.

### CODE REVIEW

File: .github/workflows/build-docker-images.yml [hunk 1]
```diff
@@ -57,3 +65,57 @@ jobs:
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./docker/transformers-pytorch-gpu
+          build-args: |
+            REF=docker-image-pt-tf
```
Comment (LysandreJik) [inline .github/workflows/build-docker-images.yml:92]: Will remove that before merging

File: .github/workflows/build-docker-images.yml [hunk 2]
```diff
@@ -57,3 +65,57 @@ jobs:
...
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./docker/transformers-tensorflow-gpu
+          build-args: |
+            REF=docker-image-pt-tf
```
Comment (LysandreJik) [inline .github/workflows/build-docker-images.yml:119]: Will remove that before merging

File: setup.py [hunk 3]
```diff
@@ -309,6 +309,37 @@ def run(self):
...
+    + extras["integrations"]
+    + extras["timm"]
+    + extras["codecarbon"]
+    + extras["quality"]
+    + extras["ja"]
+    + extras["docs_specific"]
+    + extras["sklearn"]
+    + extras["modelcreation"]
+    + extras["onnxruntime"]
+    + extras["audio"]
```
Comment (patrickvonplaten) [inline setup.py:328]:
```suggestion
```

should be fully included in `"torch-speech"`

File: setup.py [hunk 4]
```diff
@@ -309,6 +309,37 @@ def run(self):
...
+        + extras['tf']
+        + extras["sentencepiece"]
+        + extras["tokenizers"]
+        + extras["vision"]
+        + extras["quality"]
+        + extras["docs_specific"]
+        + extras["sklearn"]
+        + extras["modelcreation"]
+        + extras["onnx"]
+        + extras["audio"]
```
Comment (patrickvonplaten) [inline setup.py:341]:
```suggestion
        + extras["tf-speech"]
```

**Review by patrickvonplaten — APPROVED** [review]
Thanks a lot! Left some nits

**Pull request merged**